google-cloud-graalvm-support/pom.xml

```diff
@@ -11,4 +11,11 @@
 
   <artifactId>google-cloud-graalvm-support</artifactId>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.graalvm.nativeimage</groupId>
+      <artifactId>svm</artifactId>
+      <version>${graalvm.version}</version>
+    </dependency>
+  </dependencies>
 </project>
```

Comment:
nit: newline

Reply:
Done. Sorry I forgot to update that checkstyle to check for new lines at end of files; will send PR out shortly.